fluxion/decoders/gstreamer.py

```diff
@@ -47,6 +50,20 @@ def decode(self, input_filepath: str, output_filepath: str):
                        stdout=subprocess.DEVNULL, check=True)
         return file_checksum(output_filepath)
 
+    @lru_cache
+    def check_run(self):
+        # pylint: disable=broad-except
+        elements = self.decoder_bin.split('!')
```

Comment:
Note the element can include parameter like `h264dec ! fluvadec qos=false p1=lalala`